manifests/plugin/remote_execution/ssh.pp

```diff
@@ -0,0 +1,67 @@
+# = Foreman Proxy Remote Execution SSH plugin
```

Comment:
In that case I think it should be added to ```https://github.com/theforeman/puppet-foreman_proxy/blob/master/.sync.yml``` so we run param_docs lint checks on them as well.